### PR TITLE
Telegraf 1.18.2 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2461

add Telegraf 1.18.2 release notes

### CODE REVIEW

File: content/telegraf/v1.18/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -10,6 +10,29 @@ menu:
     parent: About the project
 ---
 
+## v1.18.2 [2021-04-29]
+
+- Make JSON format compatible with nulls to ensure Telegraf successfully detects null values and returns an empty metric without error.
+- Update `common.shim` by changing `NewStreamParser` to accept larger inputs from scanner.
```
Comment (kelseiv) [inline content/telegraf/v1.18/about_the_project/release-notes-changelog.md:16]: not sure whether common.shim is plugin?

Comment (kelseiv) [reply content/telegraf/v1.18/about_the_project/release-notes-changelog.md:16]: and/or whether it makes sense to fit lines 15-16 into a plugin update

Comment (russorat) [reply content/telegraf/v1.18/about_the_project/release-notes-changelog.md:16]: common.shim isn't a plugin, so it makes sense here.

**Review by russorat — APPROVED** [review]
LGTM

**Pull request merged**